core/storage/user/gae_models.py

```diff
@@ -131,6 +131,10 @@ class UserSubscriptionsModel(base_models.BaseModel):
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
     # IDs of feedback thread ids that this user subscribes to.
     feedback_thread_ids = ndb.StringProperty(repeated=True, indexed=True)
+    # IDs of the learners who have subscribed to this user.
+    subscriber_ids = ndb.StringProperty(repeated=True, indexed=True)
```

Comment:
I'm actually inclined to put this field in its own model type (e.g. UserSubscribersModel), since it's rather different from the other things covered by this model, which store stuff that a user subscribes to. I don't think storing it in the same model leads to any efficiency savings, either, but let me know if I've missed something?